resources/todomvc/architecture-examples/react-redux/src/reducers/todos.js

```diff
@@ -1,7 +1,17 @@
 import { ADD_TODO, DELETE_TODO, EDIT_TODO, TOGGLE_TODO, TOGGLE_ALL, CLEAR_COMPLETED } from "../constants/action-types";
 
 const initialState = [];
-const uuid = () => crypto.randomUUID();
+function uuid() {
+    let uuid = "";
+    for (let i = 0; i < 32; i++) {
+        const random = (Math.random() * 16) | 0;
+        if (i === 8 || i === 12 || i === 16 || i === 20)
+            uuid += "-";
+
+        uuid += (i === 12 ? 4 : i === 16 ? (random & 3) | 8 : random).toString(16);
```

Comment:
This line is hard to read. Why not just do this:
```js
let currentNumber = random;
if (i === 12)
    currentNumber = 4;
else if (i === 16)
    currentNumber = 8 | (random & 3);
uuid += currentNumber.toString(16);
```

Reply:
@rniwa  - I can refactor - it was just copy and paste from other implementations. 
Do you want me to make that change to all of them?

Reply:
I think we can use it for ones we're adding newly. I don't think we need to modify vue's.